app/lua53/linit.c

```diff
@@ -48,6 +48,9 @@ extern LROT_TABLE(co_funcs);
 extern LROT_TABLE(mathlib);
 extern LROT_TABLE(utf8);
 
+extern LROT_TABLE(pixbuf_map);
+extern int luaopen_pixbuf(lua_State *);
+
```

Comment:
Please also see Terrys comment https://github.com/nodemcu/nodemcu-firmware/pull/3230#issuecomment-691046239

Should we have this in one PR for Lua 5.1 and Lua 5.3?

See #3230

Reply:
I'll back the changes to this file out and we can deal with host-side deltas later.